### ISSUE TITLE
Hristov balance discussion

### ISSUE THREAD

**Issue by JoeHammad1844** [space-wizards/space-station-14#19942]
**The problem**

The hristov is barely purchased, ineffective in combat due to its low firerate and okay damage against armored targets, yes, it will do a quarter of your health if wearing sec armor but a disabler is easier to use and most other firearms will have a better chance in combat, even against single enemies, which it used to be the strongest at when first merged (although being 6TC initially was fucked and unintended)

**Solutions**

I believe there are two solutions to this
1. just give the AP back to the ammo
-This will make it essentially a better python, but only damage wise. it will still have less than half the fire rate and 5 times the weight. which you would expect for an increased price

2. change the ammo back to the first time it was buffed (60 damage, AP and stuns in one hit), require wielding for accuracy, change to 20TC and make it only available to nuclear operatives and in the surplus crate
-This will, of course, make it more powerful, but is offset by its low starting ammo, meaning you will have to purchase more ammo to get a good amount of use out of it, meaning less TC for other gadets, or you could go for war ops but that has its own issues in store for you

**Comment by B1essedknight** [comment]
Might not be my place to weigh in on this considering that I'm fairly new to this. 

I personally think that option 1 would be better as the Hristov IS a anti-material rifle and one also has to think about the fact that a magnum can kill Death Squad units but the Hristov cannot. (I dunno if we want to run with reality in this)

**Comment by Hmeister-fake** [comment]
Yeah Hristov's severely underpowered as it stands, option 1 is the best imo because in option 2 if just Nukies have it then it's gonna be like the bulldog, never used unless it's rolled in a crate.
